js/database.js

```diff
@@ -217,6 +217,44 @@
                 });
                 next();
             }
+        },
+        {
+            version: "13.0",
+            migrate: function(transaction, next) {
+                console.log('migration 13.0');
```

Comment:
Yep, we want to be sure that all of them run on startup. If one fails, we want to know where it stopped.